packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.tsx

```diff
@@ -85,6 +92,18 @@ export class ResizeGroup extends BaseComponent<IResizeGroupProps, IResizeGroupSt
   }
 
   private _onResize() {
+    // If we have some cached measurements, let's see if we can skip rendering
+    if (this._root && this._lastKnownRootWidth && this._lastKnownMeasuredWidth) {
```

Comment:
I would suggest being really careful with `shouldComponentUpdate`. If you return false at the wrong time, you can cause bugs. In this case, he's short circuiting the resize logic only, so that means the component will call `onRenderData` again in normal circumstances without having to track some variable indicating the update was the result of a resize.